### PR TITLE
Head operations for API Management entities

### PR DESCRIPTION
**Pull request by KedarJoshi**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/examples/ApiManagementHeadApiSchema.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "parameters": {
+        "serviceName": "apimService1",
+        "resourceGroupName": "rg1",
+        "api-version": "2017-03-01",
+        "subscriptionId": "subid",
+        "apiId": "57d1f7558aa04f15146d9d8a",
+		"SchemaId": "ec12520d-9d48-4e7b-8f39-698ca2ac63f1"
```
Comment (solankisamir) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/examples/ApiManagementHeadApiSchema.json]: schemaId

**Review by solankisamir — APPROVED** [review]
Looks good.

**Comment by solankisamir** [comment]
@fearthecowboy can you take a look?

**Comment by fearthecowboy** [comment]
Correct me if I'm wrong, but all these operations are just to get the entity state for the individual URIs?

Can we rename the `operationId` to `xxx_GetEntityState` instead of `xxx_Head` ?  The name `head` is not descriptive as an operation, and I'd rather like to avoid that.

**Comment by KedarJoshi** [comment]
Operations don't get entity state, rather just meta information about the entity.  In particular, use case is to check existence of entity and/or retrieve ETag header for the entity from the response.  I can rename to xxx_GetEntityTag.

**Comment by fearthecowboy** [comment]
Sure, that's fine.

**Comment by KedarJoshi** [comment]
updated with xxx_GetETag

**Comment by fearthecowboy** [comment]
@KedarJoshi  - The PR #2257 uses `TagDescription_GetEntityState` -- can we expand out `E` to `Entity` to keep these consistent in the API?

( see https://github.com/vfedonkin/azure-rest-api-specs/blob/3b1270a909de8d74bd3b783ace9ab8ab996cfe81/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimapis.json#L2052 )

**Comment by KedarJoshi** [comment]
@fearthecowboy done.

**Review by fearthecowboy — APPROVED** [review]
Looks ok to me.

**Pull request merged**